drivers/bridge/bridge.go

```diff
@@ -1009,6 +1012,10 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 		}
 	}
 
+	if err := d.storeUpdate(endpoint); err != nil {
+		logrus.Warnf("Failed to save bridge endpoint %s to store: %v", ep.id[0:7], err)
```

Comment:
Should we return error here given we will invoke necessary rollback functions on error ?